client-code/JavaScript.html

```diff
@@ -84,11 +86,17 @@
     $("#wp-go-btn").on("click", function () {
         getWPData();
     });
+    // add keydown trigger to the select element for browsers such as firefox and IE

```

Comment:
What page is the element that this key down Event handler is attached to on?

Reply:
This is for the WP lookup page

Reply:
What is the event handler for?

Reply:
This is a small update for the submit on enter functionality which fixes a small problem on Firefox and sets up the functionality (not fully) for the sub forms that exist that do not work as of yet. This definitely should be in another pull request so I can perhaps remove it from here. Sorry this pr is such a mess, I left it for 2 weeks for finals and then I was completely lost when I came back.